modules/indexing/src/main/scala/ch/epfl/bluebrain/nexus/kg/indexing/acls/AclIndexer.scala

```diff
@@ -27,26 +40,62 @@ class AclIndexer[F[_]](client: SparqlClient[F], settings: AclIndexingSettings)(i
     * @return a Unit value in the ''F[_]'' context
     */
   final def apply(event: Event): F[Unit] = event match {
-    case PermissionsCreated(path, acl, meta) =>
-      acl.acl.map { case AccessControl(id, perms) if perms(Permission.Read) => add(path, id) }
-      F.pure(())
-    case PermissionsAdded(path, identity, permissions, meta) if permissions(Permission.Read) =>
-      add(path, identity)
-    case PermissionsSubtracted(path, identity, permissions, meta) if permissions(Permission.Read) =>
+    case PermissionsCreated(path, acl, _) =>
```

Comment:
don't we meed `&& isKg(path)` here as well?

Reply:
yes. I forgot!